script/.contributors_aliases.json

```diff
@@ -261,6 +261,10 @@
     "name": "Dimitri Prybysh",
     "team": "Maintainers"
   },
+  "dmrtzn@gmail.com": {
+    "mails": ["dmrtzn@gmail.com"],
+    "name": "Daniel Mouritzen"
+  },
```

Comment:
Was it required (=crash of contributor-txt) ?

Reply:
I don't think so. I added it when I was trying to understand what `contrbutor-txt` was doing.

Reply:
Let's not bother removing it, this contributor might use another email later.